### DISCUSSION TITLE
Which nerd fonts are actually required?

### DISCUSSION THREAD

**Discussion by fira959** [sxyazi/yazi#1678] [1. Q&A] [answered]
Whenever I give yazi a try, I am having issues with the icons. Usually this is followed by me trying the different nerd fonts packages on arch before I give up and install all few hundred nerd font packages ...

It would be great to know what is actually required to display the icons, perhaps that could also be added to the install section

**Comment by sxyazi** [comment, marked as answer]
It's[ `ttf-nerd-fonts-symbols`](https://archlinux.org/packages/extra/any/ttf-nerd-fonts-symbols/), if you install Yazi through the system package manager, then it's already installed since [it's a dependency of Yazi](https://gitlab.archlinux.org/archlinux/packaging/packages/yazi/-/blob/main/PKGBUILD?ref_type=heads#L13). 

All you need to do is configure it correctly for your terminal - check the config doc for different terminals.

**Reply by fira959** [reply]
Thanks
